pageserver/src/tenant/ephemeral_file.rs

```diff
@@ -106,6 +106,19 @@ impl EphemeralFile {
 
         Ok(pos)
     }
+
+    pub(crate) async fn write_raw(
```

Comment:
nit: it's a bit unusual for write functions to return the base offset. Perhaps worth a comment?

Reply:
44679b184